includes/class-wc-stripe-webhook-state.php

```diff
@@ -182,6 +183,10 @@ public static function get_last_error_reason() {
 			return( __( 'The webhook signature was missing or was incorrectly formatted', 'woocommerce-gateway-stripe' ) );
 		}
 
+		if ( self::VALIDATION_FAILED_DUPLICATE_WEBHOOKS == $last_error ) {
+			return( __( 'Multiple webhooks are created for this site. Please remove the duplicate webhooks or re-configure the webhooks', 'woocommerce-gateway-stripe' ) );
```

Comment:
Nit: `exist` instead of `are created`? The latter gave me the impression that our side created multiple webhooks for them as a result of the error.

Reply:
Changed in [a489c7e](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3681/commits/a489c7ed6d39538cbbe3a2a08176bcc771314eba)